### PR TITLE
Release/0.0.2

### PR DESCRIPTION
**Pull request by kubaplas**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,23 @@
 
 Changelog for `cargo-odra`.
 
+## [0.0.2] - 2022-08-12
```
Comment (kpob) [inline CHANGELOG.md:5]: Shouldn't be 0.1.0?

File: src/backend.rs [hunk 2]
```diff
@@ -55,12 +78,38 @@ impl Backend {
...
+
+        let name = add.name.as_ref().unwrap().clone();
+
+        if conf.backends.is_none() {
+            backends = HashMap::new();
+        } else {
+            backends = conf.backends.unwrap();
+        }
+
+        if let Vacant(e) = backends.entry(name) {
```
Comment (kpob) [inline src/backend.rs:98]: Is it `e` for `entry`?

File: src/backend.rs [hunk 3]
```diff
@@ -211,28 +282,22 @@ impl Backend {
             });
         }
 
+        let name = add.name.clone().unwrap();
```
Comment (kpob) [inline src/backend.rs]: Is it always non-optional? Is safe to just uwrap?

File: .github/workflows/ci-cargo-odra.yml [hunk 4]
```diff
@@ -21,7 +21,7 @@ on:
 jobs:
   build_and_test:
     name: Build and test
-    runs-on: ubuntu-latest
+    runs-on: buildjet-8vcpu-ubuntu-2204
```
Comment (kpob) [inline .github/workflows/ci-cargo-odra.yml]: Maybe some comments on it, what is it, or a link? For the next generations.

**Review by kpob — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (kubaplas) [reply CHANGELOG.md:5]: Cargo Odra is not connected to Odra release cycle. I decide when 0.1.0 is ready.
This release is made mostly for the bugfix.

File: src/backend.rs (hunk 2 above)
Comment (kubaplas) [reply src/backend.rs:98]: It seems so. This was suggested by clippy, so I left it that way

File: src/backend.rs (hunk 3 above)
Comment (kubaplas) [reply src/backend.rs]: At this point it will always be non-optional. Unless we forget about it and reuse the method... I'll fix it.

Comment (kubaplas) [reply src/backend.rs]: Fixed

**Pull request merged**